### DISCUSSION TITLE
French translation review needed

### DISCUSSION THREAD

**Discussion by seb-troiani** [Eugeny/tabby#5491] [General]
Hi there
I just finished the draft translation for French language.
Since some strings were hard to find within the app, I did my best but some corrections will probably be needed.
**Please review at https://translate.tabby.sh/project/tabby/fr**
Kind regards from France and thanks to Eugene for this app !
Stay safe

**Comment by Eugeny** [comment]
Thank you for your work! I understand only very basic French so it'd be great if someone could proofread it and upvote the translations. You can also add comments to strings where the context is unclear. 

@allcontributors please add @seb-troiani for translations
